src/Symfony/Bridge/Twig/Extension/FormExtension.php

```
@@ -95,6 +103,62 @@ public function getTests()
     }
 
     /**
+     * @internal
+     */
+    public function __get($name)
+    {
+        if ('renderer' === $name) {
+            @trigger_error(sprintf('Using the "%s::$renderer" property is deprecated since version 3.2 as it will be removed in 4.0.', __CLASS__), E_USER_DEPRECATED);
+
+            if (is_array($this->renderer)) {
+                $renderer = $this->renderer[0]->get($this->renderer[1]);
+                if (isset($this->renderer[2])) {
+                    $renderer->setEnvironment($this->renderer[2]);
+                }
+                $this->renderer = $renderer;
+            }
+        }
+
+        return $this->$name;
+    }
+
+    /**
+     * @internal
+     */
+    public function __set($name, $value)
+    {
+        if ('renderer' === $name) {
```

we should throw an exception for any other name